src/rohit/Arraybasics.java

```diff
@@ -0,0 +1,66 @@
+package rohit;
+
+public class Arraybasics {
+	int minimumnumber(int[] arr1) {
+		int min = arr1[0];
+		for (int i = 0; i < arr1.length; i++) {
+			if (min > arr1[i]) {
+				min = arr1[i];
+			}
+		}
+		return min;
+	}
+
+	int maximumnumber(int[] arr2) {
+
+		int max = arr2[0];
+		for (int i = 0; i < arr2.length; i++) {
+			if (max < arr2[i]) {
+				max = arr2[i];
+			}
+		}
+		return max;
+	}
+
+	int averageofnumbers(int[] arr3) {
+		int sum = 0;
+		for (int i = 0; i < arr3.length; i++) {
+			sum = sum + arr3[i];
+		}
+		int avg = sum / arr3.length;
+		return avg;
+	}
+
+	void displayallnumbers0tolength(int[] arr4) {
+		System.out.println("Array listing from 0 to length- ");
+		for (int i = 0; i < arr4.length; i++) {
+			System.out.println(arr4[i]);
+		}
+	}
+
+	void displayallnumberslengthto0(int[] arr5) {
+		System.out.println("Array listing form length to 0- ");
+		for (int i = arr5.length - 1; i >= 0; i--) {
+			System.out.println(arr5[i]);
+		}
+	}
+
+	int averageofminmaxnumbers(int max, int min) {
```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, so this expecting two variable in method looks wrong as per the implementation perspective, user will give us array, not min & max value. I hope you got my point.

You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (minimumnumber(input)+maxnumber(input))/2